roles/tas_single_node/tasks/podman.yml

```diff
@@ -34,18 +58,7 @@
       "{{ tas_single_node_tsa_enabled }}",
       "true",
     ]
-  loop:
-    - "{{ tas_single_node_fulcio_server_image }}"
-    - "{{ tas_single_node_trillian_log_server_image }}"
-    - "{{ tas_single_node_trillian_logsigner_image }}"
-    - "{{ tas_single_node_rekor_image }}"
-    - "{{ tas_single_node_ct_server_image }}"
-    - "{{ tas_single_node_redis_image }}"
-    - "{{ tas_single_node_trillian_db_image }}"
-    - "{{ tas_single_node_tuf_image }}"
-    - "{{ tas_single_node_netcat_image }}"
-    - "{{ tas_single_node_nginx_image }}"
-    - "{{ tas_single_node_tsa_image }}"
+  loop: "{{ images_list }}"
```

Comment:
I'm not sure that this change is correct. For example, with your change the Redis image will be the 10th item in the images list and thus will be pulled when `"{{ tas_single_node_tsa_enabled }}"` if I'm looking right.

Both the trillian_db_image and redis_image are already in the list. We just need to update the conditions for them to get pulled when they should be.